pkg/docker/volume.go

```diff
@@ -10,6 +10,12 @@ import (
 )
 
 func (d *Docker) ListVolumes(filter opts.FilterOpt) ([]*types.Volume, error) {
+	if d.GetClient() == nil {
```

Comment:
Good catch but I think the main fix is to actually connect the client in `cluster/ampagent/cmd/uninstall.go`
```
	dockerCli := docker.NewDockerCli(stdin, stdout, stderr)
```
To
```
	dockerCli := docker.NewDockerCli(stdin, stdout, stderr)
	if err := Docker.Connect(); err != nil {
		return err
	}
```

Reply:
right, I'll leave a check and return an error is the client is not connected.